src/functions/get-filtered-data.js

```diff
@@ -0,0 +1,21 @@
+const getFilteredData = (data, categoryBy, clear) => {
+  if (categoryBy && categoryBy === "FILTER_BY_PAINTING") {
```

Comment:
It would be better if constants are in the constant folder

Reply:
Ok will look into that